model/model.go

```diff
@@ -150,6 +151,14 @@ type AppConfig struct {
 	Maintenance    bool            `key:"maintenance" constraint:"(?i)^(true|false)$"`
 	SSLConfig      *SSLConfig      `key:"ssl"`
 	Nginx          *NginxAppConfig `key:"nginx"`
+	ProxyLocations []string        `key:"proxyLocations"`
+	ProxyDomain    string          `key:"proxyDomain"`
+	Locations      []*Location
+}
+
+type Location struct {
```

Comment:
Locations are simply PathPatterns for an App defined in the AppConfig so why don't we just call them that or am I missing something?

Reply:
Can we add something in the README.md above about what the difference between ProxyLocations and ProxyDomain? Maybe add an example of an app with two paths, `/ui` and `/api` and how the ProxyLocations and ProxyDomain needs to defined.

Reply:
I don't remember exactly why we gave such names to `proxyLocations` and `proxyDomains`, most probably we followed `nginx` config terms. Each application in `deis` has one or multiple `domain`s. E.g. we created `app1` and `app2` so by default all the traffic coming to `app1.example.com` will be routed to k8s service of the `app1` and all the traffic coming to `app2.example.com` will be routed to k8s service of the `app2`.
We want path `app1.example.com/webhooks` to be always handled by `app2`. So we set up annotation for k8s service of the `app2`:
```
$ kubectl describe svc app2 -n app2
Name:              app2
Namespace:         app2
Labels:            app=app2
                   router.deis.io/routable=true
Annotations:       
                   router.deis.io/domains=app2
                   router.deis.io/maintenance=False
                   router.deis.io/proxyDomain=*app1*
                   router.deis.io/proxyLocations=/webhooks
...
```

Reply:
Great to have the documentation example. And let's continue with the naming since they are the nginx standard. :100: :1st_place_medal: